src/app.module.ts

```diff
@@ -13,8 +12,10 @@ import { UserModule } from "./user/user.module";
 @Module({
   imports: [
     ConfigModule.forRoot({ isGlobal: true }),
+    MongooseModule.forRoot(process.env.DATABASE_URL),
     HealthModule,
     CatsModule,
+    TilesModule,
     MongooseModule.forRoot(process.env.DATABASE_URL),
```

Comment:
MongooseModule.forRoot(process.env.DATABASE_URL)
Twice?

Reply:
Thanks for reminding me.